package.json

```diff
@@ -44,7 +44,7 @@
   "dependencies": {
     "@material-ui/core": "^4.12.3",
     "@material-ui/icons": "^4.11.2",
-    "@truffle/hdwallet-provider": "^1.4.3",
+    "@truffle/hdwallet-provider": "1.5.0",
```

Comment:
Was it intentional that all dependencies are fixed to a particular version instead of setting a minimum version? We still have the lock file to get reproducible builds and not silently change versions.